libminifi/include/core/state/Value.h

```diff
@@ -580,6 +573,10 @@ struct SerializedResponseNode {
   SerializedResponseNode(const SerializedResponseNode &other) = default;
 
   SerializedResponseNode &operator=(const SerializedResponseNode &other) = default;
+
+  bool empty() {
```

Comment:
we could make it const

Reply:
changed it in [7b80c81](7b80c812be57906faff88b091302fa36227f940f)